diff --git a/includes/ckeditor.page.inc b/includes/ckeditor.page.inc
index 03f79e8..f727cd4 100644
--- a/includes/ckeditor.page.inc
+++ b/includes/ckeditor.page.inc
@@ -286,7 +286,7 @@ function ckeditor_filter_xss() {
       }
       continue;
     }
-    if (function_exists($filters[$name]['prepare callback'])) {
+    if (isset($filters[$name]['prepare callback']) && function_exists($filters[$name]['prepare callback'])) {
       $text = $filters[$name]['prepare callback']($text, $format_filters[$name], $format, '', TRUE, $cache_id);
     }
     $text = $filters[$name]['process callback']($text, $format_filters[$name], $format, '', TRUE, $cache_id);
